src/components/VisibilityCard/VisibilityCard.tsx

```diff
@@ -50,45 +57,57 @@ const useStyles = makeStyles(
 interface VisibilityCardProps {
   children?: React.ReactNode | React.ReactNodeArray;
   data: {
+    availableForPurchase?: string;
+    isAvailableForPurchase?: boolean;
     isPublished: boolean;
     publicationDate: string;
+    visibleInListings?: boolean;
```

Comment:
I get a feeling that we should create new component that would be embedding `VisibilityCard` passing additional elements as props